### PR TITLE
Add transaction allocations to credit memo

### PR DESCRIPTION
**Pull request by Boydbueno**

### CODE REVIEW

**Comment by ashkarpetin** [comment]
please update changelog

**Comment by Boydbueno** [comment]
@Rebilly/php please review

**Comment by Boydbueno** [comment]
@Rebilly/php please review

**Review by ashkarpetin — APPROVED** [review]

File: src/Entities/AllocationCollection.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    /**
+     * @param array $data
+     *
+     * @return array|TransactionAllocation[]
+     */
+    public function createTransactions(array $data)
+    {
+        return array_map(function ($item) {
```
Comment (nanobash) [inline src/Entities/AllocationCollection.php:56]:
```suggestion
        return array_map(static function ($item) {
```

**Review by nanobash — APPROVED** [review]

File: src/Entities/AllocationCollection.php (hunk 1 above)
Comment (Boydbueno) [reply src/Entities/AllocationCollection.php:56]: I can pick this up as a separate PR for the whole project.

Comment (ashkarpetin) [reply src/Entities/AllocationCollection.php:56]: Please leave it without static.

Comment (nanobash) [reply src/Entities/AllocationCollection.php:56]: May I know why @ashkarpetin?

Comment (nanobash) [reply src/Entities/AllocationCollection.php:56]: and also why don't we use return type declarations?

**Pull request merged**